lms/djangoapps/lti_provider/views.py

```diff
@@ -149,4 +154,26 @@ def render_courseware():
     :return: an HttpResponse object that contains the template and necessary
     context to render the courseware.
     """
-    return HttpResponse('TODO: Render refactored courseware view.')
+    usage_id = lti_params['usage_id']
+    course_id = lti_params['course_id']
+    course_key = CourseKey.from_string(course_id)
+    user = request.user
+    course = get_course_with_access(user, 'load', course_key)
+    staff = has_access(request.user, 'staff', course)
+    instance, _ = get_module_by_usage_id(request, course_id, usage_id)
+
+    fragment = instance.render('student_view', context=request.GET)
+
+    context = {
+        'fragment': fragment,
+        'course': course,
+        'disable_accordion': True,
+        'allow_iframing': True,
+        'disable_header': True,
+        'disable_footer': True,
+        'disable_tabs': True,
+        'staff_access': staff,
+        'xqa_server': settings.FEATURES.get('USE_XQA_SERVER', 'http://xqa:server@content-qa.mitx.mit.edu/xqa'),
```

Comment:
I'm a little wary about having `mit.edu` hardcoded as a default, even if it's intended to be overridden. People in the Open edX community might enable this feature accidentally without configuring the URL, and there's a risk that it will just work -- which sounds like a good thing, until you realize that it could potentially put a lot of extra load on MIT's service, and MIT may not be prepared to support backwards-compatibility for this service forever. It would be better to have this raise an exception if this feature is turned on without a valid URL for this setting, with a clear, friendly exception message explaining what needs to be done to turn this service on. That's the route that we took for the [Harvard's `annotation_storage_url` setting](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/videoannotation_module.py#L51-L56).